cl/scrapers/transformer_extractor_utils.py

```diff
@@ -0,0 +1,186 @@
+import json
+import os
+from tempfile import NamedTemporaryFile
+
+import requests
+from django.conf import settings
+from django.core import serializers
+
+
+def document_extract(path=None, file_content=None, do_ocr=False):
+    """Extract document content.
+
+    :param path:
+    :param file_content:
+    :param do_ocr:
+    :return:
+    """
+    service = "%s/%s" % (settings.BTE_URL, "extract_doc_content")
+    if path is not None:
+        with open(path, "rb") as file:
+            file_content = file.read()
+    elif file_content is None:
+        return {"err": "File not supplied"}
+
+    if path is not None:
+        file_name = os.path.basename(path)
+    else:
+        file_name = os.path.basename(file_content.name)
+    return requests.post(
+        url=service,
+        files={"file": (file_name, file_content)},
+        params={"do_ocr": do_ocr},
+        timeout=60 * 60,
+    ).json()
+
+
+def serialize_audio_object(af):
+    """
+
+    :param af: Audio File Object
+    :return: Convert audio w/ metadata
+    """
+    af_dict = json.loads(serializers.serialize("json", [af]))[0]["fields"]
+    docket_dict = json.loads(serializers.serialize("json", [af.docket]))[0][
+        "fields"
+    ]
+    court_dict = json.loads(serializers.serialize("json", [af.docket.court]))[
+        0
+    ]["fields"]
+    af_dict["docket"] = docket_dict
+    af_dict["docket"]["court"] = court_dict
+    with NamedTemporaryFile(suffix=".json") as tmp:
+        with open(tmp.name, "w") as json_data:
+            json.dump(af_dict, json_data)
+        with open(tmp.name, "rb") as file:
+            af = file.read()
+    return af
```

Comment:
I think this will be obviated by doing the POST correctly, but there's a lot going on here. I guess I'll just note two things:

1. You can always do `object.__dict__()` to convert them to a dict. I think that's easier than the hoops you're going through here.

2. You're creating JSON data from a Python object be `json.dump`ing it to disk, and then reading it from disk. Check out `json.dumps`, where the `s` is short for string (I believe). It's the cousin of `loads`. 

Apologies if I'm missing something here.